lib/form_data.js

```diff
@@ -76,6 +77,19 @@ FormData.prototype.append = function(field, value, options) {
 
   // pass along options.knownLength
   this._trackLength(header, value, options);
+
+  if (!this._entryList[field]) {
+    this._entryList[field] = [];
+  }
+  this._entryList[field].push(value);
+};
+
+FormData.prototype.get = function(field) {
+  return this._entryList[field] ? this._entryList[field][0] : null;
+};
+
+FormData.prototype.getAll = function(field) {
+  return this._entryList[field] || [];
```

Comment:
From the spec
> The getAll(name) method, when invoked, must return the values of all entries whose name is name, in order, from the context object’s entry list, and the empty list otherwise.

Do a `Array.filter` in the list.